catalog/spatial/registry/registry-admin-modules/registry-admin-poller-service/src/main/java/org/codice/ddf/registry/admin/poller/AdminPollerServiceBean.java

```diff
@@ -200,9 +211,40 @@ public boolean sourceStatus(String servicePID) {
     }
 
     @Override
-    public List<Map<String, Object>> allSourceInfo() {
+    public List<List<Map<String, Object>>> allSourceInfo() {
```

Comment:
After looking at this I think we might want to go ahead and do the source to registry entry mapping in this method. Since we have both list of both registry metacards and source maps it should be pretty easy to match them up using the registry-id from the metacard and the registry-id from the source metatype properties and then just return a list of maps as before. So the top level list of maps would look something like
```[
   {
       source { .....} //single source
       registry { ....} //single associated registry entry
   }
]```